scripts/mentorProfile.js

```diff
@@ -25,30 +25,61 @@ async function initializeInfo() {
 function showInfo(dataMentors) {
   // armazena as áreas de nome e categoria da tela
   let mentorName = document.getElementById("mentor-name-profile");
-  let category = document.getElementById("category-profile");
-  let skill = document.getElementById("skill-profile")
+  let category_1 = document.getElementById("category-1");
+  let category_2 = document.getElementById("category-2");
+  let category_1Rating = document.getElementById("category1-rating");
+  let category_2Rating = document.getElementById("category2-rating");
+
+  // // todas as estrelas do HTML
+  const stars_1 = document.querySelectorAll(".stars_1");
+  const stars_2 = document.querySelectorAll(".stars_2");
+
+  let classStars_2 = document.getElementById("classStar_2");
+  let rating_2;
+  let evaluation_2;
+  let dataValue_2;
 
   // vai receber o id do mentor
   mentorName.textContent = dataMentors.name;
-  category.textContent = dataMentors.email;
-  skill.textContent = dataMentors.skills.skill
-  
+  category_1.textContent = dataMentors.Categories[0].name;
+  category_1Rating.textContent = dataMentors.Categories[0].name;
+
+ 
+  // PARA CATEGORIA 1
   // mostrar a avaliação da habilidade
-  let rating = parseFloat(dataMentors.skills.rating);
+  let rating_1 = dataMentors.Categories[0].UserCategories.rating;
```

Comment:
Podemos usar const, já que esse varíavel não é reassignada.

```suggestion
  const rating_1 = dataMentors.Categories[0].UserCategories.rating;
```

Reply:
fixed